packages/example/src/pages/index.tsx

```diff
@@ -34,6 +36,17 @@ export default function Index() {
           type="checkbox"
         />
       </label>
+
+      <hr/>
+      <p> after clicking on: <a href="#sync">a hash link</a> the url should contain the hash despite changing params</p>
+      <hr/>
+
+      <p>the router query object provided by next should contain all query params:</p>
+      <pre>
+        {JSON.stringify(router.query, null, 2)}
```

Comment:
This doesn't seem to be changing for me.

Reply:
Have you tried rebuilding the `next-query-params` package? 

```
yarn build
```

Reply:
On the otherside you'll probably understand why I created this PR a little bit more :)

Reply:
🤦‍♂️ Of course, after rebuilding this works – thanks! I think I need a coffee …

Reply:
haha same here, as I forgot the `eslint --fix`